src/index.html

```diff
@@ -7,12 +7,57 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="Imac"
+        class="card_image card_image--imac"
+      />
+      <div class="description">
+        Apple A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="product_code">Product code: 195434</div>
+      <div class="star_review">
+        <div class="stars_active"></div>
+        <div class="stars_active"></div>
+        <div class="stars_active"></div>
+        <div class="stars_active"></div>
```

Comment:
It seems like you have multiple divs with the class 'stars_active'. If these are meant to represent individual stars in a rating system, consider using a more semantic approach, such as using a list or a span with appropriate ARIA attributes for better accessibility.